File: 0034-find-first-and-last-position-of-element-in-sorted-array/solution.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
        int left=0;
        int right=nums.length-1;
        int m=-1;
        int n=-1;
       for(int i=0;i<nums.length;i++){
        if(nums[i]==target){
            m=i;
            break;
        }
       }
       for(int i=nums.length-1;i>=0;i--){
        if(nums[i]==target){
            n=i;
            break;
        }
       }
        return new int[]{m,n};
        
    }
}

File: 0034-find-first-and-last-position-of-element-in-sorted-array/solution.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        l1=[]
        for i in range(len(nums)):
            if nums[i]==target:
                l1.append(i)
        if l1:
            return [l1[0],l1[-1]]
        return [-1,-1]
